common.env

```diff
@@ -1 +1,2 @@
 DEFAULT_CHANNEL_SLUG=default-channel
+HTTP_IP_FILTER_ALLOW_LOOPBACK_IPS=True
```

Comment:
I recommend indicating that it shouldn't be blindly copied into a production environment:

```suggestion
# Note: recommended value is False for production usage,
# refer to documentation for more details.
HTTP_IP_FILTER_ALLOW_LOOPBACK_IPS=True
```